oracle-plugin/src/e2e-test/java/io.cdap.plugin/oracle/runners/TestRunnerRequired.java

```diff
@@ -26,7 +26,9 @@
 @CucumberOptions(
   features = {"src/e2e-test/features"},
   glue = {"stepsdesign", "io.cdap.plugin.common.stepsdesign", "io.cdap.plugin.oracle.stepsdesign"},
-  tags = {"@Oracle_Required"},
+  tags = {"@Oracle_Required and not @PLUGIN-1535"},
```

Comment:
Why do we need this change ?

Reply:
Made the change for the same bug(1535) issue, as @Oracle_Required tag is added at the feature level.

Reply:
Same here. We can remove the tag for the bug now